lib/rex/text/rand.rb

```diff
@@ -258,5 +258,27 @@ def self.rand_mail_address
       mail_address << '@'
       mail_address << Rex::Text.rand_hostname
     end
+
+    # Generate a strong password of specified length and attributes
+    def self.rand_password(len=10, uppercase=true, lowercase=true, numbers=true, special=false)
+      foo=[]
```

Comment:
I wonder if there's a more descriptive variable name that could be used? 👀

Reply:
Just spotted that the existing methods use `foo` 😅  Happy for this to be left as-is or renamed
```suggestion
      allowed_characters = []
```